.ci/install_qemu.sh

```diff
@@ -37,4 +39,9 @@ sudo -E make install
 # in a repository.
 sudo ln -sf $(command -v qemu-system-$(arch)) /usr/bin/
 
+if [ "$ID" == "centos" ]; then
+	qemu_path="/usr/local/bin/qemu-system-$(arch)"
+	sudo ln -sf "$qemu_path" /usr/bin/
```

Comment:
I think this is not needed anymore as we are already creating the symlink above.  See L40.

Reply:
@chavafg centos does not retrieve a value with $(command -v qemu-system-$(arch))